mlstream/utils.py

```diff
@@ -20,7 +20,9 @@ def create_h5_files(data_root: Path,
                     basins: List,
                     dates: List,
                     forcing_vars: List,
-                    seq_length: int):
+                    seq_length: int,
+                    forcings_file_format: str,
+                    concat_static: bool = True):
```

Comment:
Why do we need `concat_static` here? I don't see a usage in the method.

Reply:
`create_h5_files` function constructs an object of `LumpedBasin` type which has a parameter for `concat_static`. This parameter is defaulted to `True` and I included this to have the option of creating an object `LumpedBasin` type with a `False` `concat_static` if need be. 
if `create_h5_files` will never call `LumpedBasin` with `concat_static` as False then I guess this can be removed.

Reply:
I see now. We create LumpedBasins here, but we only extract the forcings from them, not the static attributes. Since LumpedBasin has `with_attributes==False` by default, `concat_static` has no effect anyways. 
Maybe we can make this more explicit: Don't have `concat_static` as a method parameter here, but set `with_attributes=False` in `dataset = LumpedBasin(...)` below, with a comment like "We only store time-series forcings in the .h5-file, so we don't need static attributes."

Reply:
Changes made.